src/DeveloperDoesNotExistException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Drupal\apigee_edge;
```

Comment:
Introduce a new `Exception` namepsace.

Reply:
And there is an `Exception` folder for custom exception classes under `src`.